modules/social_features/social_tagging/social_tagging.install

```diff
@@ -121,3 +121,22 @@ function social_tagging_update_8004() {
     \Drupal::logger('social_tagging')->info($e->getMessage());
   }
 }
+
+/**
+ * Create, if not set, default settings to enable tagging on content.
+ */
+function social_tagging_update_11001(): void {
+  // Get the editable social tagging configuration.
+  $tag_settings = \Drupal::getContainer()->get('config.factory')->getEditable('social_tagging.settings');
+
+  // Get the multiple types of nodes.
+  /** @var \Drupal\node\NodeTypeInterface[] $node_types */
+  $node_types = \Drupal::service('entity_type.manager')->getStorage('node_type')->loadMultiple();
```

Comment:
There can be a possibility in an already existing environment that more content types (node types) than what we expect are already present. This will ignore our intended node types, i.e, book, event, landing_page, page, and topic.

Also, we cannot assume that it's true for all. This is a little ambiguous decision to make it happen on already existing environment. May be can only focus on fresh installations?

Reply:
Hi Navneet, thank you for your comment. 

Can you please elaborate a bit more over the first comment? The idea of the hook update is to create settings for content types that don't have a valid settings entry. 

About the update for existing installations, when you go to the settings page the default value is "true", this is why I added this hook update with the default value as "enabled".

Please let me know your thoughts.

Reply:
This is for already existing environments, which don't have this problem. It is only on the first installation. Site managers might have also customized it according to their needs. So, it is safe to fix it only for "first installation".

Reply:
Remove this hook update